tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -305,6 +305,15 @@
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ oc_header }}
-    oc wait --for condition=Ready osdpns/openstack --timeout=40m
-  # TODO: work on network configuration for making possible to run this task on other IP ranges
-  when: "edpm_node_ip.startswith('192.168.122')"
+    oc get pod -l app=openstackansibleee
+    NODESET_NAME=openstack
```

Comment:
please update the docs to keep this in sync with tests

Reply:
the docs already mention how to follow the dataplane status: https://openstack-k8s-operators.github.io/data-plane-adoption/openstack/edpm_adoption/#post-checks
what we do here is just a trick to get the last service to ensure all the services are completed